webserver/views/api/v1/dataset_eval.py

```diff
@@ -4,6 +4,8 @@
 from webserver.decorators import auth_required
 from webserver.views.api import exceptions
 import db.dataset_eval
+import db.dataset
+from . import datasets
```

Comment:
It might be better to do an absolute import there. http://programmers.stackexchange.com/a/159505/36419